pkg/ebpf/finding.go

```diff
@@ -107,7 +107,7 @@ func getCType(t interface{}) string {
 	}
 }
 
-func getMetadataFromSignatureMetadata(sigMetadata detect.SignatureMetadata) *trace.Metadata {
+func getMetadataFromSignatureMetadata(sigMetadata detect.SignatureMetadata, triggerEvent trace.Event) *trace.Metadata {
```

Comment:
What's the idea of putting this on metadata and not directly in `getArguments`?

Reply:
I was trying to not change the actual arguments today, so I thought it would be best o pass as properties. But i can change to args, what you think?

Reply:
I think it is ok to add it to the arguments, since this data is missing today